lib/laa_crime_schemas/types/types.rb

```diff
@@ -89,12 +89,15 @@ module Types
                                     ])
 
     OutgoingsType = String.enum(*%w[
-                                  housing
+                                  rent
+                                  mortgage
+                                  board_and_lodging
                                   council_tax
                                   childcare
                                   maintenance
-                                  legal_aid
+                                  legal_aid_contribution
                                 ])
+
     HOUSING_PAYMENT_TYPES = %w[
```

Comment:
Are we then deprecating this?

Reply:
The OutgoingsType will stay I think. The individual type names I've copied from what @RoseSAK is working on.

Reply:
I meant the HOUSING_PAYMENT_TYPES, sorry